src/dev/test_generator/generate/optimizer.py

```
@@ -0,0 +1,226 @@
+        parsed = self._parser.parse(source_path)
+        request = GenerationRequest(parsed_module=parsed, config=config)
+        request.start()
+
+        context = self._gather_context(source_path)
+
+        try:
+            if context is not None and self._fabric is not None:
+                self._process_module_level(request, source_path, context)
+            else:
+                self._process_per_entity(request, context)
+        except Exception:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)